Util benchmark suite stopped building on nightly Rust

Probably for a week now, and using `rustc 1.12.0-dev (1d5b758ba 2016-08-14)` for the sake of this report, util has stopped compiling for me:

```rust
error[E0425]: unresolved name `U512`
  --> benches/bigint.rs:54:4
   |
54 |                    U512([
   |                    ^^^^

error[E0425]: unresolved name `U512`
  --> benches/bigint.rs:60:25
   |
60 |                            old.overflowing_sub(U512([p, p, p, p, p, p, p, new])).0
   |                                                ^^^^

error[E0425]: unresolved name `U512`
  --> benches/bigint.rs:70:15
   |
70 |            (0..n).fold(U512([0, 0, 0, 0, 0, 0, 0, 0]),
   |                        ^^^^

error[E0425]: unresolved name `U512`
  --> benches/bigint.rs:71:37
   |
71 |                    |old, new| { old.overflowing_add(U512([new, new, new, new, new, new, new, new])).0 })
   |                                                     ^^^^

error[E0531]: unresolved variant or struct `U512`
  --> benches/bigint.rs:90:9
   |
90 |                            let U512(ref u512words) = old.full_mul(U256([rand::random::<u64>(), rand::random::<u64>(), rand::random::<u64>(), rand::random::<u64>()]));
   |                                ^^^^

error[E0425]: unresolved name `U128`
   --> benches/bigint.rs:101:15
    |
101 |           (0..n).fold(U128([12345u64, 0u64]), |old, new| { old.overflowing_mul(U128::from(new)).0 })
    |                       ^^^^

error[E0433]: failed to resolve. Use of undeclared type or module `U128`
   --> benches/bigint.rs:101:72
    |
101 |           (0..n).fold(U128([12345u64, 0u64]), |old, new| { old.overflowing_mul(U128::from(new)).0 })
    |                                                                                ^^^^^^^^^^ Use of undeclared type or module `U128`

error: no method named `overflowing_add` found for type `ethcore_util::U256` in the current scope
  --> benches/bigint.rs:37:132
   |
37 |            (0..n).fold(U256([rand::random::<u64>(), rand::random::<u64>(), rand::random::<u64>(), rand::random::<u64>()]), |old, new| { old.overflowing_add(U256::from(new)).0 })
   |                                                                                                                                             ^^^^^^^^^^^^^^^
   |
   = help: items from traits can only be used if the trait is in scope; the following trait is implemented but not in scope, perhaps add a `use` for it:
   = help: candidate #1: `use ethcore_util::Uint`

error: no method named `overflowing_sub` found for type `ethcore_util::U256` in the current scope
  --> benches/bigint.rs:45:132
   |
45 |            (0..n).fold(U256([rand::random::<u64>(), rand::random::<u64>(), rand::random::<u64>(), rand::random::<u64>()]), |old, new| { old.overflowing_sub(U256::from(new)).0 })
   |                                                                                                                                             ^^^^^^^^^^^^^^^
   |
   = help: items from traits can only be used if the trait is in scope; the following trait is implemented but not in scope, perhaps add a `use` for it:
   = help: candidate #1: `use ethcore_util::Uint`

error: no method named `overflowing_mul` found for type `ethcore_util::U256` in the current scope
  --> benches/bigint.rs:79:132
   |
79 |            (0..n).fold(U256([rand::random::<u64>(), rand::random::<u64>(), rand::random::<u64>(), rand::random::<u64>()]), |old, new| { old.overflowing_mul(U256::from(new)).0 })
   |                                                                                                                                             ^^^^^^^^^^^^^^^
   |
   = help: items from traits can only be used if the trait is in scope; the following trait is implemented but not in scope, perhaps add a `use` for it:
   = help: candidate #1: `use ethcore_util::Uint`

error: aborting due to 3 previous errors

Build failed, waiting for other jobs to finish...
````

 plus a few warnings:
````
warning: unused result which must be used, #[warn(unused_must_use)] on by default
  --> benches/trie.rs:75:4
   |
75 |                    t.insert(&i.0, &i.1);
   |                    ^^^^^^^^^^^^^^^^^^^^^

warning: unused result which must be used, #[warn(unused_must_use)] on by default
   --> benches/trie.rs:114:4
    |                               
114 |                   t.insert(&i.0, &i.1);
    |                   ^^^^^^^^^^^^^^^^^^^^^

warning: unused result which must be used, #[warn(unused_must_use)] on by default
   --> benches/trie.rs:152:4
    |
152 |                   t.insert(&i.0, &i.1);
    |                   ^^^^^^^^^^^^^^^^^^^^^

warning: unused result which must be used, #[warn(unused_must_use)] on by default
   --> benches/trie.rs:187:4
    |
187 |                   t.insert(&i.0, &i.1);
    |                   ^^^^^^^^^^^^^^^^^^^^^

warning: unused result which must be used, #[warn(unused_must_use)] on by default
   --> benches/trie.rs:224:4
    |
224 |                   t.insert(&i.0, &i.1);
    |                   ^^^^^^^^^^^^^^^^^^^^^

warning: unused result which must be used, #[warn(unused_must_use)] on by default
   --> benches/trie.rs:261:4
    |
261 |                   t.insert(&i.0, &i.1);
    |                   ^^^^^^^^^^^^^^^^^^^^^

error: Could not compile `ethcore-util`.
````

--- a/util/benches/bigint.rs
+++ b/util/benches/bigint.rs
@@ -24,11 +24,11 @@
 #![feature(asm)]
 
 extern crate test;
-extern crate ethcore_util;
+extern crate bigint;
 extern crate rand;
 
 use test::{Bencher, black_box};
-use ethcore_util::U256;
+use bigint::uint::{U256, U512, Uint, U128};
 
 #[bench]
 fn u256_add(b: &mut Bencher) {

